### ISSUE TITLE
训练V3版本的模型ch_PP-OCRv3_rec.yml报错，acc一直是0

### ISSUE THREAD

**Issue by hds1999** [PaddlePaddle/PaddleOCR#11902]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：X86 windows11 
- 版本号/Version：Paddle：paddle-gpu 2.5.1  PaddleOCR：PaddleOcrV2.7 问题相关组件/Related components：ch_PP-OCRv3_rec.yml
- Global:
  debug: false
  use_gpu: true
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_ppocr_v3
  save_epoch_step: 10
  eval_batch_step: [0, 100]
  cal_metric_during_train: true
  pretrained_model: ./pretrain_models/ch_rec/best_accuracy.pdparams
  checkpoints:
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 100
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05

Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform:
  Backbone:
    name: MobileNetV1Enhance
    scale: 0.5
    last_conv_stride: [1, 2]
    last_pool_type: avg
    last_pool_kernel_size: [2, 2]
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 64
            depth: 2
            hidden_dims: 120
            use_guide: True
          Head:
            fc_decay: 0.00001
      - SARHead:
          enc_dim: 512
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - SARLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ch_rec/
    ext_op_transform_idx: 1
    label_file_list:
    - ./train_data/ch_rec/train_list.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 32 
    drop_last: true
    num_workers: 1
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ch_rec/
    label_file_list:
    - ./train_data/ch_rec/eval_list.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 32
    num_workers: 1
- 运行指令/Command Code：python tools/train.py -c configs\rec\PP-OCRv3\ch_PP-OCRv3_rec.yml -o Global.use_gpu=True Global.save_model_dir=./output/inference   Train.loader.batch_size_per_card=1 
- 完整报错/Complete Error Message：
[2024/04/09 16:36:40] ppocr INFO: train with paddle 2.5.1 and device Place(gpu:0)
[2024/04/09 16:36:40] ppocr INFO: Initialize indexs of datasets:['./train_data/ch_rec/train_list.txt']
list index out of range
[2024/04/09 16:36:40] ppocr INFO: Initialize indexs of datasets:['./train_data/ch_rec/eval_list.txt']
[2024/04/09 16:36:43] ppocr INFO: train dataloader has 103 iters
[2024/04/09 16:36:43] ppocr INFO: valid dataloader has 103 iters
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.conv1._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.conv1._batch_norm.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.conv1._batch_norm.bias not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.conv1._batch_norm._mean not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.conv1._batch_norm._variance not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._depthwise_conv._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._depthwise_conv._batch_norm.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._depthwise_conv._batch_norm.bias not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._depthwise_conv._batch_norm._mean not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._depthwise_conv._batch_norm._variance not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._pointwise_conv._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._pointwise_conv._batch_norm.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._pointwise_conv._batch_norm.bias not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._pointwise_conv._batch_norm._mean not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.0._pointwise_conv._batch_norm._variance not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._depthwise_conv._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._depthwise_conv._batch_norm.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._depthwise_conv._batch_norm.bias not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._depthwise_conv._batch_norm._mean not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._depthwise_conv._batch_norm._variance not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._pointwise_conv._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._pointwise_conv._batch_norm.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._pointwise_conv._batch_norm.bias not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._pointwise_conv._batch_norm._mean not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.1._pointwise_conv._batch_norm._variance not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.2._depthwise_conv._conv.weight not in model
[2024/04/09 16:36:43] ppocr WARNING: The pretrained params Teacher.backbone.block_list.2._depthwise_conv._batch_norm.weight not in model

**Comment by GreatV** [comment]
是不是把预训练模型载入错了

**Comment by hds1999** [comment]
> 是不是把预训练模型载入错了
用的配置文件：
https://github.com/PaddlePaddle/PaddleOCR/blob/main/configs/rec/PP-OCRv3/ch_PP-OCRv3_rec.yml
预训练模型：
https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_rec_train.tar

**Comment by TingquanGao** [comment]
使用 https://github.com/PaddlePaddle/PaddleOCR/blob/main/configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml 训练配置，或者：
参考文档 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/knowledge_distillation.md#215-%E8%92%B8%E9%A6%8F%E6%A8%A1%E5%9E%8B%E5%BE%AE%E8%B0%83 把 https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_rec_train.tar 中的student参数提取出来。

**Comment by TingquanGao** [comment]
长时间未回复，该issue已关闭，如仍有问题可以reopen或新开issue。
